assign2/category.js

```diff
@@ -0,0 +1,69 @@
+import { HISTORY_LIST } from "./DATA.js";
+
+// 로컬스토리지에 저장
+const category = HISTORY_LIST.map((list) => {
+  const { title, getto } = list;
+  const obj = {
+    category: title,
+    isPlus: getto,
+  };
+  return obj;
+});
+
+function saveToLocalStorage(data) {
+  window.localStorage.setItem("key", JSON.stringify(data));
+}
+saveToLocalStorage(category);
+
+const plusInput = document.getElementById("plus_input");
+const minusInput = document.getElementById("minus_input");
+const plusCategory = document.getElementById("plus");
+const minusCategory = document.getElementById("minus");
+
+const getItem = window.localStorage.getItem("key");
+const getCategory = JSON.parse(getItem);
+
+getCategory.forEach((ele) => {
+  const { category, isPlus } = ele;
+  const h3 = document.createElement("h3");
+  h3.innerText = category;
+  isPlus ? plusCategory.appendChild(h3) : minusCategory.appendChild(h3);
+});
+
+plusInput.addEventListener("keypress", (e) => {
+  isEntered(e, true);
+});
+
+minusInput.addEventListener("keypress", (e) => {
+  isEntered(e, false);
+});
+
+function isEntered(e, isPlus) {
+  if (e.keyCode === 13) {
+    const h3 = document.createElement("h3");
+    h3.innerText = e.target.value;
```

Comment:
13이 무엇을 의미하는지 모를 수도 있으니까 
```suggestion
function isEntered(e, isPlus) {
if (event.key === "Enter") {
    const h3 = document.createElement("h3");
    h3.innerText = e.target.value;
```
요런식이나 13을 상수화해서 표현하는 것도 가독성 측면에서 좋을 것 같아!!!

Reply:
아 그러네 모를수도 있겠다....